react/src/base/inputs/Date.stories.js

```diff
@@ -31,55 +31,103 @@ which you may need to remove before submitting the form.
 };
 
 export const dateInput = () => (
-  <SprkTextInput
-    label="Date"
-    name="date"
-    placeholder="01/01/2019"
-  />
+  <SprkInputContainer>
+    <SprkLabel htmlFor="date-1">Date</SprkLabel>
+    <SprkInput id="date-1" placeholder="01/01/2021" />
+  </SprkInputContainer>
 );
 
 dateInput.story = {
   name: 'Default',
   parameters: {
-    jest: [
-      'SprkTextInput',
-    ]
+    jest: ['SprkInput', 'SprkLabel', 'SprkInputContainer'],
   },
 };
 
 export const invalidDateInput = () => (
-  <SprkTextInput
-    label="Date"
-    name="date"
-    placeholder="01/01/2019"
-    valid={false}
-    errorMessage="There is an error on this field."
-  />
+  <SprkInputContainer>
+    <SprkLabel htmlFor="date-2">Date</SprkLabel>
+    <SprkInput
+      id="date-2"
+      placeholder="01/01/2021"
+      isValid={false}
+      ariaDescribedBy="invalid-error"
+    />
+    <SprkFieldError id="invalid-error">
+      <SprkIcon
+        iconName="exclamation-filled"
+        additionalClasses="sprk-b-ErrorIcon"
+        aria-hidden="true"
```

Comment:
Is this a prop?

Reply:
Not currently a prop. Have contacted Steph about creating an issue to add it.

Reply:
Cool, thanks!